components/UniversalGlossary.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div class="universal-glossary__container">
+    <bx-data-table>
+      <bx-table>
+        <bx-table-head>
+          <bx-table-header-row class="universal-glossary__table-header">
+            <bx-table-header-cell>Notation</bx-table-header-cell>
+            <bx-table-header-cell>Description</bx-table-header-cell>
+          </bx-table-header-row>
+        </bx-table-head>
+        <bx-table-body>
+          <bx-table-row
+            v-for="{ notation, description } in Object.values(glossary)"
+            :key="notation"
+            class="universal-glossary__table-row"
```

Comment:
```suggestion
            class="universal-glossary__notation"
```
I think it defines better what this container has.

Reply:
Thanks! Incorporated in https://github.com/Qiskit/saiba/pull/407/commits/5111904e75418a99eb6cbd280df0a10ec0a4144e